frontend/src/routes/(app)/project/[owner]/[project]/+layout.server.ts

```diff
@@ -26,39 +34,26 @@ export async function load({ cookies, params, url }) {
 	if (!project) {
 		throw error(404, 'Could not load project config');
 	}
-	const slices = await ZenoService.getSlices(project.uuid);
-	if (!slices) {
-		throw error(404, 'Could not load slices');
-	}
-	const columns = await ZenoService.getColumns(project.uuid);
-	if (!columns) {
-		throw error(404, 'Could not load columns');
-	}
-	const models = await ZenoService.getModels(project.uuid);
-	if (!slices) {
-		throw error(404, 'Could not load models');
-	}
-	const metrics = await ZenoService.getMetrics(project.uuid);
-	if (!metrics) {
-		throw error(404, 'Could not load metrics');
-	}
-	const folders = await ZenoService.getFolders(project.uuid);
-	if (!folders) {
-		throw error(404, 'Could not load folders');
-	}
-	const tags = await ZenoService.getTags(project.uuid);
-	if (!tags) {
-		throw error(404, 'Could not load tags');
-	}
+
+	const requests = [
+		ZenoService.getSlices(project.uuid),
+		ZenoService.getColumns(project.uuid),
+		ZenoService.getModels(project.uuid),
+		ZenoService.getMetrics(project.uuid),
+		ZenoService.getFolders(project.uuid),
+		ZenoService.getTags(project.uuid)
+	];
+
+	const results = await Promise.all(requests);
 
 	return {
 		project: project,
-		slices: slices,
-		columns: columns,
-		models: models,
-		metrics: metrics,
-		folders: folders,
-		tags: tags,
+		slices: results[0] as Slice[],
+		columns: results[1] as ZenoColumn[],
+		models: results[2] as string[],
+		metrics: results[3] as Metric[],
+		folders: results[4] as Folder[],
+		tags: results[5] as Tag[],
```

Comment:
Might be a bit problematic to find the reason if something fails but probably fine.